PSSwagger/PSSwagger.psm1

```diff
@@ -800,6 +822,70 @@ function New-PSSwaggerModule {
             -PSHeaderComment $PSHeaderComment
     }
 
+    if ($CopyUtilityModuleToOutput) {
+        Write-Verbose -Message $LocalizedData.CopyingUtilityModule
+        $utilityModuleInfo = Get-Module PSSwaggerUtility
```

Comment:
>Get-Module PSSwaggerUtility [](start = 29, length = 27)

Since PSSwaggerUtility module is a nested module in PSSwagger.psd1, this will not return the expected module info unless user explicitly imports it.

Reply:
In what case is the expected module not returned? I tried having the utility module installed globally and not, and in both cases the expected module was returned.